masterlist.yaml

```diff
@@ -270,11 +270,20 @@ plugins:
       - 'DLCCoast.esm'
       - 'DLCworkshop02.esm'
       - 'DLCworkshop03.esm'
+  - name: 'DLCUltraHighResolution.esm'
+    global_priority: -120
+    after:
+      - 'DLCRobot.esm'
+      - 'DLCworkshop01.esm'
+      - 'DLCCoast.esm'
+      - 'DLCworkshop02.esm'
+      - 'DLCworkshop03.esm'
+      - 'DLCNukaWorld.esm'
 
   ### Patches
   - name: 'Unofficial Fallout 4 Patch.esp'
     url:
-      - 'http://afkmods.iguanadons.net/index.php?/files/file/1765-unofficial-fallout-4-patch/'
+      - 'https://afkmods.iguanadons.net/index.php?/files/file/1765-unofficial-fallout-4-patch/'
```

Comment:
This seems like it's semantically a different change than any of the others. Perhaps it could be in its own commit?